### PR TITLE
Added validation for create campaign

### LINKED ISSUE #37: Create Campaign validation
1. Incorrect token address should show meaningful message
2. media/logo/website links should check for valid http/https link
3. all links must not return ERR_CERT_AUTHORITY_INVALID.

### LINKED ISSUE #53: One of the images in test data is missing in deployed app.
image

### PR DESCRIPTION
**Pull request by UsamaSaleem14**

### CODE REVIEW

File: src/hooks/useCampaignsHandler.ts [hunk 1]
```diff
@@ -31,6 +33,39 @@ export const useCampaignsHandler = () => {
   const [campaigns, setCampaigns] = useState<ICampaign[]>([]);
 
   const addCampaign = async () => {
+    if (
```
Comment (TBomberman) [inline src/hooks/useCampaignsHandler.ts:36]: Can we put these into a separate validation function?

File: src/utils/ContractUtils.ts [hunk 2]
```diff
@@ -101,3 +101,9 @@ export const getCampaignKey = (projectToken: string) => {
   const hash = ethers.utils.keccak256(encodedParams);
   return hash;
 };
+
+export const checkLinkValidity = (link: string): boolean => {
+  const pattern =
```
Comment (TBomberman) [inline src/utils/ContractUtils.ts:106]: Is it possible to use this? https://gabrieleromanato.name/nodejs-check-the-validity-of-a-website-ssl-certificate

File: scripts/campaigns.json [hunk 3]
```diff
@@ -40,7 +40,7 @@
     {
         "name": "Mr Blinky Bling DEFCON 26 Indie Badge",
         "description": "We're back and this time we're making a social game using blockchain (yes, really!) and badges! Shall we play a game? So we've been trolling you all on Twitter about this whole Blockchain badge concept and the general reaction has been: They must be trolling!",
-        "logoLink": "http://mrblinkybling.com/img/profile.png",
+        "logoLink": "https://www.google.com/images/errors/robot.png",
         "websiteLink": "http://mrblinkybling.com/",
```
Comment (TBomberman) [inline scripts/campaigns.json]: Also need to change the website link.
Is the website link being validated?

File: src/utils/ContractUtils.ts (hunk 2 above)
Comment (UsamaSaleem14) [reply src/utils/ContractUtils.ts:106]: No the package is used for node js application that is for backend.

File: scripts/campaigns.json (hunk 3 above)
Comment (UsamaSaleem14) [reply scripts/campaigns.json]: We are validating if the website link is in correct format or not

File: src/hooks/useCampaignsHandler.ts (hunk 1 above)
Comment (UsamaSaleem14) [reply src/hooks/useCampaignsHandler.ts:36]: Done

File: scripts/campaigns.json (hunk 3 above)
Comment (TBomberman) [reply scripts/campaigns.json]: http://mrblinkybling.com/ <- this is not a secure website, need to be httpS instead of http.

Comment (UsamaSaleem14) [reply scripts/campaigns.json]: Changed

**Review by TBomberman — APPROVED** [review]

**Pull request merged**